components/portal/messages/controllers.js

```diff
@@ -438,11 +438,11 @@ define(['angular'], function(angular) {
 
     .controller('AnnouncementsController', ['$q', '$log', '$filter',
       '$sessionStorage', '$scope', '$rootScope', '$document', '$sanitize',
-      '$mdDialog', 'miscService',
-      'messagesService', 'PortalAddToHomeService', 'MISC_URLS',
+      '$mdDialog', 'miscService', 'messagesService', 
```

Comment:
:art: `441:53  error  Trailing spaces not allowed  no-trailing-spaces`

https://travis-ci.org/uPortal-Project/uportal-app-framework/jobs/340695770#L720

Reply:
Fixed.